create-ladder.js

```diff
@@ -26,4 +26,40 @@ const addHandlersToElements = () => {
     })(i);
   }
 }
+
+const addStep = document.getElementById('addStep');
```

Comment:
Ok, this file is getting quite big, right now everything defined in this file is in global namespace.. This is very bad practice and should be avoided, sorry I missed this before.

You should wrap this file in a single function, then have that function expose anything required to be in the global scope.